src/index/ivf_raft/ivf_raft_config.h

```diff
@@ -64,6 +70,12 @@ class RaftIvfPqConfig : public IvfPqConfig {
     CFG_STRING internal_distance_dtype;
     CFG_FLOAT preferred_shmem_carveout;
     KNOHWERE_DECLARE_CONFIG(RaftIvfPqConfig) {
+        KNOWHERE_CONFIG_DECLARE_FIELD(k)
+            .set_default(10)
+            .description("search for top k similar vector.")
+            .set_range(1, 1024)
+            .for_search();
+
```

Comment:
The upper bound value 1024 is limited by hardware. However, this limitation can be easily broken by optimizing software, which can reduce the explanation cost.

Reply:
Good Idead.